benchexec/tablegenerator/util.py

```diff
@@ -180,10 +180,13 @@ def format_options(options):
     return '<span style="display:block">' + '</span><span style="display:block">'.join(line for line in lines if line.strip()) + '</span>'
 
 def to_decimal(s):
-    # remove whitespaces and trailing units (e.g., in '1.23s')
     if s:
-        s, _ = split_number_and_unit(s.strip())
-        return Decimal(s) if s else None
+        if s.lower() == 'nan':
+            return Decimal(s)
+        else:
```

Comment:
Missing case for infinity here?

Reply:
Yes, this lead to statistics ignoring (+-) infinity.

Reply:
But that is not correct. If there is `+Inf` in the values, of course the max value and the average in the statistics must be `+Inf`. Median and min should still have proper values, though.

Reply:
It should already be fixed in 691d0bd61 . I forgot to add '(erroneously)' :wink: